config/settings.yml

```diff
@@ -88,7 +88,7 @@ active_features:
 reject_refuse_messaging_released_at: <%= DateTime.new(2018, 4, 24, 23, 05, 0).change(offset: '+100') %>
 
 # agfs_fee_reform_release_date: <%= Date.new(2018, 4, 1) %>
-agfs_fee_reform_release_date: <%= ENV["ENV"].eql?('api-sandbox') ? Date.new(2018, 1, 1) : Date.new(2018, 4, 1) %>
+agfs_fee_reform_release_date: <%= ENV["ENV"].in?(['api-sandbox','demo','development']) ? Date.new(2018, 1, 1) : Date.new(2018, 04, 1) %>
```

Comment:
**[not a blocker/nit pick]** Rubocop 👮‍♀️: Spaces after commas, or just turn it into a `%w[]` array format instead.